src/pudl/metadata/fields.py

```diff
@@ -2138,6 +2171,15 @@
             },
         }
     },
+    "plant_parts_eia": {
+        "energy_source_code_1": {
```

Comment:
The `energy_source_code_1` and `prime_movers_eia` resource override enum constraints could maybe just be moved to the general entry in `FIELD_METADATA`

Reply:
In the database tables these are both constrained by virtue of their FK relationships to the `energy_sources_eia` and `prime_movers_eia` coding tables, so adding the enums to the field definitions would be duplicative. But for the purpose of imposing those constraints on this free floating output table it seems like the ENUM constraint makes sense.

Reply:
Ah right, makes sense.